PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeFlowCallback.java

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api;
+
+import androidx.fragment.app.FragmentActivity;
+
+/**
+ * Callback for receiving success and error when calling
+ * {@link PayPalNativeClient#tokenizePayPalAccount(FragmentActivity, PayPalRequest, PayPalNativeFlowCallback)}
+ */
+//TODO: i changed the name because i think one call back is enough. The clients wants to know whether it succeeded or failed
+public interface PayPalNativeFlowCallback {
+
+    /**
+     * @param success if the paypal experience is successful.
+     * @param error an exception that occurred while initiating PayPal Native
+     */
+    void onResult(PaymentMethodNonce success, Exception error);
```

Comment:
```suggestion
    void onResult(PayPalAccount nonce payPalAccountNonce, Exception error);
```

This callback should only be returning PayPalAccountNonces